src/OpenRoad.tcl

```diff
@@ -110,6 +110,98 @@ proc write_def { args } {
   ord::write_def_cmd $filename $version
 }
 
+# partition
+sta::define_cmd_args "partition_design" { [-max_num_macro max_num_macro] \
+                                          [-min_num_macro min_num_macro] \
+                                          [-max_num_inst max_num_inst] \
+                                          [-min_num_inst min_num_inst] \
+                                          [-net_threshold net_threshold] \
+                                          [-virtual_weight virtual_weight] \
+                                          [-rpt_file rpt_file] \
+                                        }
+proc partition_design { args } {
+    sta::parse_key_args "partition_design" args keys {-max_num_macro -min_num_macro
+                     -max_num_inst  -min_num_inst -net_threshold -virtual_weight -rpt_file} flags {  }
+    if { [info exists keys(-rpt_file)] } {
```

Comment:
this says the rpt_file arg is required but the doc string in define_cmd_args says it is optional.
I think it is a mistake to make it required. Is there some reason it doesn't go to the console like a normal command?